store/src/java/com/zimbra/cs/service/admin/AdminDocumentHandler.java

```diff
@@ -1033,4 +1036,29 @@ public Server verifyServerPerms(ServerSelector serverSelector, ZimbraSoapContext
         checkRight(zsc, server, Admin.R_getServer);
         return server;
     }
+
+    protected boolean checkPasswordSecurity(Map<String, Object> context) throws ServiceException {
```

Comment:
Was this method copied from somewhere? Would it be worthwhile to put it into the parent DocumentHandler instead?

Reply:
Thanks added in DocumentHandler